pkg/build-rpm.sh

```
@@ -0,0 +1,72 @@
+# Release assets are named with the full tag version (e.g. 1.0.0-beta3);
+# RAG_SERVER_VERSION is the suffix-stripped value the spec expects in SOURCES.
+TAG_VERSION="${RAG_SERVER_BRANCH#v}"
+ARTIFACT_DIR="${ARTIFACT_DIR:-$(pwd)/release-artifacts}"
+RELEASE_URL="https://github.com/pgEdge/pgedge-rag-server/releases/download/${RAG_SERVER_BRANCH}"
+RAW_URL="https://raw.githubusercontent.com/pgEdge/pgedge-rag-server/${RAG_SERVER_BRANCH}"
+
+prepare() {
+  setup_dnf_build_env
+
+  echo "Copying packaging files..."
+  cp ${COMPONENT_NAME}/rpm/pgedge-rag-server.spec ~/rpmbuild/SPECS/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)